Sources/Core/OACompoundIconUtils.mm

```diff
@@ -87,7 +87,7 @@ @implementation OACompoundIconUtils
         return result;
 
     // color filled background icon
-    NSString *backgroundIconName = [NSString stringWithFormat:@"ic_bg_point_%@_center%@", shapeName, sizeName];
+    NSString *backgroundIconName = [OANativeUtilities getScaledResourceName:[NSString stringWithFormat:@"ic_bg_point_%@_center%@", shapeName, sizeName]];
```

Comment:
What are these resources? It's better to transfer them and take them from the assets